doc/codingGuidelines.md

```diff
@@ -187,6 +194,8 @@ Headers should be included in the following order, with each section separated b
 #include <mayaUsd/fileio/shading/shadingModeRegistry.h>
```

Comment:
Maybe add a group with a pxr header or two just before the mayaUsd ones for completeness?

Reply:
Agreed that we should probably add a pxr header to help clarify... will hold off on doing that until we make a decision [here](https://github.com/Autodesk/maya-usd/pull/400#discussion_r401844482), however.

Reply:
Sounds good. I'm on board for option **B**.